tutorials/embedding/__init__.py

```diff
@@ -17,5 +17,3 @@
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
-
-import gen_word2vec
```

Comment:
Is deleting this line the right thing to do here? I could also call `tf.load_op_library`. Just wasn't sure how this is being used.